indico/modules/events/abstracts/forms.py

```diff
@@ -390,30 +390,38 @@ class CreateEmailTemplateForm(EditEmailTemplateRuleForm):
                      "You can customize it later."))
 
 
-class AbstractForm(IndicoForm):
+class AbstractAttachmentForm(IndicoForm):
+    attachments = FileField(_('Attachments'), param_name='attachments', multiple_files=True)
+
+    def __init__(self, *args, **kwargs):
+        self.abstract = kwargs.pop('abstract', None)
+        self.event = kwargs.pop('event')
+        super(AbstractAttachmentForm, self).__init__(*args, **kwargs)
+
+
+class AbstractForm(AbstractAttachmentForm):
```

Comment:
I'd rather make it a mixin - it's weird that the abstract form inherits from an attachment form IMO